core/src/main/java/org/mskcc/cbio/portal/model/StructuralVariant.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright (c) 2017 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mskcc.cbio.portal.model;
+
+public class StructuralVariant {
+
+    private Integer geneticProfileId;
+    private Long internalId;
+    private Integer sampleIdInternal;
+    private String sampleId;
+    private Long site1EntrezGeneId;
+    private String site1HugoSymbol;
+    private String site1EnsemblTranscriptId;
+    private Integer site1Exon;
+    private String site1Chromosome;
+    private Integer site1Position;
+    private String site1Description;
+    private Long site2EntrezGeneId;
+    private String site2HugoSymbol;
+    private String site2EnsemblTranscriptId;
+    private Integer site2Exon;
+    private String site2Chromosome;
+    private Integer site2Position;
+    private String site2Description;
+    private String site2EffectOnFrame;
+    private String dnaSupport;
+    private String rnaSupport;
+    private Integer normalReadCount;
+    private Integer tumorReadCount;
+    private Integer normalVariantCount;
+    private Integer tumorVariantCount;
+    private Integer normalPairedEndReadCount;
+    private Integer tumorPairedEndReadCount;
+    private Integer normalSplitReadCount;
+    private Integer tumorSplitReadCount;
+    private String annotation;
+    private String breakpointType;
+    private String center;
+    private String connectionType;
+    private String eventInfo;
+    private String variantClass;
+    private Integer length;
```

Comment:
ah...here it is a number....in DB is was not. Here it is correct :+1:  (but make it `int` instead of `Integer`)

Reply:
Done